doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Beaglebone/index.md

```diff
@@ -5,4 +5,111 @@ permalink: /:path/
 sort_idx: 30
 ---
 
-SDK integration instructions for select platforms
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The guide provides information on how to **cross-compile** C++ SDK for Beaglebone or Beaglebone Black.
+Alternatively, you can build the Kaa C++ endpoint SDK right on the Beaglebone board.
+Refer to [the Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/) for further details.
+
+## Install dependencies
+
+**The further instructions must be executed on the host machine.**
+
+**Verified against:**
+
+ - **Host OS:** Ubuntu 14.04 64-bit LTS
+ - **Target OS:** Ubuntu 16.04 32-bit LTS
+
+1. Install build prerequisites.
+
+        sudo apt-get install cmake build-essentials
+
+1. Download toolchain.
+
+        mkdir bb_root && cd bb_root || exit
```

Comment:
Remove `|| exit` parts. It might be good for scripts, but almost useless for user guides.